pkgs/tools/misc/kb/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "kb";
+  version = "0.1.7";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "gnebbia";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0hjvf1ycwf4f8rnrshj30ij56mkji2ic3ibpschyvng5jslh1h9b";
+  };
+
+  preBuild = ''
```

Comment:
If we are patching source files contained in the tarball that are not generated in configurePhase, than this should be done in postPatch